python/rmm/_cuda/stream.pyx

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from libc.stdint cimport uintptr_t
+from libcpp cimport bool
+
+from rmm._lib.cuda_stream_view cimport cuda_stream_view
+from rmm._lib.lib cimport (
+    cudaStream_t,
+    cudaStreamDefault,
+    cudaStreamLegacy,
+    cudaStreamPerThread,
+)
+
+import cupy
+from numba import cuda
+
+from rmm._lib.cuda_stream import CudaStream
+
+
+cdef class Stream:
+    def __init__(self, obj=None):
+        """
+        BA Stream represents a CUDA stream.
```

Comment:
What is `BA Stream`?

Reply:
Intentionally introduced typo when playing with pre-commit and forgot :)